safe-global/safe-wallet-web
# feat: use isTestnet flag

## What this PR changes
Uses new `isTestnet` flag in ChainConfig instead of hardcoded list of testnet shortnames.

## How to test it
- Open the Network selector
- Check that all testnets are correctly partitioned

## Checklist
